org.qt_project.qtcreator.yaml

```diff
@@ -0,0 +1,91 @@
+app-id: org.qt_project.qtcreator
+runtime: org.kde.Sdk
+runtime-version: "6.2"
+sdk: org.kde.Sdk
+command: qtcreator
+# Qt Creator already provides desktop launchers and AppData metadata
+# but since we cannot use the original AppID, we have to rename the original files
+rename-desktop-file: org.qt-project.qtcreator.desktop
+rename-appdata-file: org.qt-project.qtcreator.appdata.xml
+rename-icon: QtProject-qtcreator
+finish-args:
+# Qt Creator crashes GNOME Shell 3.36 on Wayland
+# - --socket=wayland
+# - --socket=fallback-x11
+- --socket=x11
+- --share=ipc
+- --share=network
+- --device=dri
+- --filesystem=host
+cleanup:
+- /include
+- /lib/arch
+- /lib/cmake
+- /lib/libQt6Core5Compat.prl
+- /lib/metatypes
+- /lib/qtcreator/objects-RelWithDebInfo
+- /lib/x86_64-linux-gnu
+- /mkspecs
+- /modules
+- /qml
+- /share/cmake
+- /share/pkgconfig
+- "*.a"
+cleanup-commands:
+- mv /app/lib/*-linux-gnu/* /app/lib/
+
+modules:
+# addition Qt modules that are not part of the KDE SDK
+- name: qt5compat
+  sources:
+  - type: git
+    url: git://code.qt.io/qt/qt5compat.git
+    tag: v6.2.1
```

Comment:
This is a nitpick, but the usual convention is that `sources` is the last element of module.

Reply:
I didn't know that. My preference is that the name and source of a module are close together because this is the most important information about the software that makes up the flatpak. Everything about the buildsystem is exchangable.